lib/classes/Variables.js

```diff
@@ -344,11 +344,24 @@ class Variables {
     }, valueToPopulate);
   }
 
-  warnIfNotFound(variableString, valueToPopulate) {
+  deleteNullEnvironmentVariables() {
```

Comment:
@pmuens Currently the variable evaluation is done in the package phase, so it should be long there for now, imo.